wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -49,36 +55,29 @@ def setUp(self):
             slug="local.adur.churchill.2021-05-06",
         )
         self.post = PostFactory()
-        self.post_election = PostElectionFactory(election=self.election, post=self.post)
+        self.post_election = PostElectionFactory(
+            election=self.election, post=self.post
+        )
+
+    def test_zero_candidates(self):
+        response = self.client.get(
+            self.post_election.get_absolute_url(), follow=True
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "elections/post_view.html")
+        # self.assertContains(response, "No candidates known yet.")
```

Comment:
@michaeljcollinsuk I'm not able to get line 68 and 83 to pass. Is there a better way to target the partial? Or do I need render it in the test explicitly?

Reply:
If I add:
`self.assertTemplateUsed(response, "elections/includes/_post_meta_title.html")`
before line 68, the test passes. The same strategy doesn't work for `test_num_candidates`

Reply:
Unless perhaps it's a problem with the string 🤔

Reply:
This is where I'd drop in to an interactive debugger (`ipdb` to see what's going on in the response. Or you can just use `print` on `response.content` and see what it produces. Don't forget to run `pytest` with `-s` to tell it not to capture the output.

I think the response object also has a `templates` list against it - one to try with `ipdb`.

Reply:
Hmm that is odd as everything looks correct.. if you add in a `import pdb; pdb.set_trace()` and then check `response.content` you should see the HTML that is rendered, including the meta title, what does that look like?